DigitalLearningSolutions.Data/Models/User/CentreUserDetails.cs

```diff
@@ -2,14 +2,6 @@
 {
     public class CentreUserDetails

```

Comment:
Can we rename this? It's disturbingly close to UserCentreDetails. Something like ChooseACentreAccount...?